src/integrations/settings-integration.php

```diff
@@ -255,6 +256,10 @@ public function register_hooks() {
 			\add_action( 'admin_init', [ $this, 'register_setting' ] );
 			\add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
 			\add_action( 'in_admin_header', [ $this, 'remove_notices' ], \PHP_INT_MAX );
+
+			// Remove the post types and taxonomies made public notifications (if any).
+			$this->remove_post_types_made_public_notification();
+			$this->remove_taxonomies_made_public_notification();
```

Comment:
Is this safe to execute right away?

Reply:
Yes, the notification center basically don't care if the notification you want to remove exists or not 😁 
If it doesn't exits, it silently returns.